source/SpinalHDL/Libraries/stream.rst

```diff
@@ -534,6 +534,27 @@ This util take its input stream and routes it to ``outputCount`` stream in a seq
      outputCount = 3
    )
 
+StreamTransactionExtender
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This util helps to extend one input transaction with specified ``count``+1 times of output transactions with the same payload.
+The ``count`` signal would be captured and registered while inputStream fire.
+
```

Comment:
A ``count`` value of zero would result in no additional payload insertion causing the input and output transactions to be identical.

Reply:
Question: the `count` that is captured, is it captured once, or at every inputStream payload valid signal ?  That should be stated.  Maybe like:

The ``count`` is captured and registered each time inputStream fires for an individual payload.